source/preparing/text.rst

```diff
@@ -19,6 +19,16 @@ depth about the features of Atom's Git integration. If you're interested in addi
 
 **Atom** can be downloaded for free from https://atom.io/.
 
+Visual Studio Code
+~~~~~~~~~~~~~~~~~~
+
+Visual Studio Code is a source code editor developed by Microsoft for Windows, Linux and macOS. It includes support for debugging, 
```

Comment:
Please keep line lengths 120 characters or below, to avoid wrapping. (It's one of our guidelines).